python/amici/sbml_import.py

```diff
@@ -987,7 +987,8 @@ def _process_log_likelihood(self,
                 )(obs_id), locals=dict(zip(
                     _get_str_symbol_identifiers(obs_id),
                     (obs_id, obs['measurement_symbol'], sigma_id)
-                )))
+                ))),
+                'dist': noise_distributions.get(str(obs_id), 'normal'),
```

Comment:
`noise_distributions.get(str(obs_id), 'normal')` occurs already a few lines above. reuse result?

Reply:
would require unpacking the dict comprehension.

Reply:
true. okay. not expecting `noise_distributions` to grow too large...

Reply:
Decided to unpacking that monster of a comprehension, probably more readable like that anyways.